src/plugins/custom_import_map/test/jest.config.js

```diff
@@ -2,7 +2,7 @@
  * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
  */
-
+process.env.TZ = 'UTC';
```

Comment:
want to confirm that this file is executed by some other process and all vars are set and this is exact equivalent of previous syntax TZ=UTC

Reply:
This apply only for test. Don't have full understanding of how this value is being used in other places tbh but at least all tests are passed.

Reply:
This PR is already merged in main and 2.x branch. If there is any doubt, we can create a new issue and work on a new thread.